skore/src/skore/sklearn/cross_validation/cross_validation_reporter.py

```diff
@@ -124,6 +124,7 @@ def __init__(self, *args, **kwargs):
         self.estimator = args[0] if len(args) >= 1 else kwargs.get("estimator")
         self.X = args[1] if len(args) >= 2 else kwargs.get("X")
         self.y = args[2] if len(args) == 3 else kwargs.get("y")
+        self.cv = kwargs.get("cv")
```

Comment:
If this parameter can't be positional, please specify it in the docstring.

Reply:
It's a kwarg here because it's a kwarg in `sklearn.cross_validate`. I don't really know how to document this; in `sklearn.cross_validate` they show the actual argument list and so they have a `*`, but for us the arguments are `*args, **kwargs`... Would you have an idea of how to do this?